ereol_app_feeds/src/Feed/CategoriesFeed.php

```diff
@@ -40,7 +40,12 @@ public function getData() {
               'title' => $wrapper->field_picked_title->value(),
               'type' => 'sub_category',
               'query' => ParagraphHelper::VALUE_NONE,
-              'attachment' => isset($paragraphData['list']) ? $paragraphData['list'] : ParagraphHelper::VALUE_NONE,
+              'attachment' => !empty($paragraphData['list'])
+              ? [
```

Comment:
Think it would be more readable if the `?` was on first line

Reply:
I don't agree (and usually follow https://google.github.io/styleguide/javaguide.html#s4.5.1-line-wrapping-where-to-break). However, I've rewritten to `if (…) { … }` for better readability.